tests/setup/setup-window-globals.js

```diff
@@ -0,0 +1,6 @@
+// Check if we're in a JSDOM test or not
```

Comment:
Minor note: is this meant to be a TODO?

Reply:
It's just a doc comment for `global.window` check.